features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -1532,6 +1532,56 @@ ALTER TABLE IDN_CONFIG_FILE ADD CONSTRAINT RESOURCE_ID_FILE_FOREIGN_CONSTRAINT F
 IDN_CONFIG_RESOURCE (ID) ON DELETE CASCADE ON UPDATE RESTRICT
 /
 
+CREATE TABLE IDN_REMOTE_FETCH_CONFIG(
```

Comment:
Formatting

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/3143